front/app/components/HookForm/InputMultilocWithLocaleSwitcher/index.tsx

```diff
@@ -34,10 +34,14 @@ const InputMultilocWithLocaleSwitcher = ({ name, ...rest }: Props) => {
   );
 
   // Select the first error messages from the field's multiloc validation error
-  const errorMessage = Object.values(
+  const validationError = Object.values(
```

Comment:
`Select the first error messages from the field's multiloc validation error`
Minor: I noticed that we are doing this in many places. Do you think this is something we could move to a function? A util maybe that we can reuse and change in one place 🤔

Reply:
Good point - there is indeed a lot of repetition with the error handling across components. Part of the reason is that I want to leave it flexible and not couple it too much to start with, as we are perhaps not aware of all the use cases yet and might want to handle these errors differently in the different components at some point. Once we gain confidence in the abstraction - we can dry it out 👍